src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String getRandomColor() {
```

Comment:
please pay attention to the checklist: "Don’t use static methods in your solution"

Reply:
I deleted 'static' and created object class insted of that.